src/Playwright/API/Generated/Enums/ReducedMotion.cs

```diff
@@ -0,0 +1,48 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Playwright
+{
+    public enum ReducedMotion
+    {
+        [EnumMember(Value = "reduce")]
+        Reduce,
+        [EnumMember(Value = "no-preference")]
+        NoPreference,
+        [EnumMember(Value = "null")]
+        Null,
```

Comment:
My two cents here. With this `Null`  I would wonder why would we have a `Null` element instead of making the enum nullable. 
`None` could be a common word for this. https://github.com/search?l=C%23&q=org%3Adotnet+None&type=Code

Reply:
The enum is already nullable, we need a tri-state semantics here with Null resetting to the defaults. You can do None instead of Null, just make sure it serializes to "null" in the channel.

Reply:
Yeah, I remember we had that same thought for some other enums. I'd have `none` to avoid confusion.